README.md

```diff
@@ -1,206 +1,68 @@
-To support user deployment flows, signing an OCI Artifact will not change the `@digest` or `artifact:tag` reference.
-To support content movement across multiple certification boundaries, artifacts and their signatures will be easily copied within and across [OCI conformant][oci-distribution-conformance] registries.
+| File Name     | Description |
+| -------- | ----------- |
+| [ADA-notation-security-audit-23.pdf](./security/reports/audit/ADA-notation-security-audit-23.pdf) | Security audit report in 2023 covering [notation](https://github.com/notaryproject/notation), [notation-go](https://github.com/notaryproject/notation-go), and [notation-core-go](https://github.com/notaryproject/notation-core-go) repositories |
+| [ADA-fuzzing-audit-22-23.pdf](./security/reports/fuzzing/ADA-fuzzing-audit-22-23.pdf) | Fuzz testing audit in 2023 covering [notary](https://github.com/notaryproject/notaty), [notation-go](https://github.com/notaryproject/notation-go), and [notation-core-go](https://github.com/notaryproject/notation-core-go) repositories |
 
-![Notary v2 dependent projects](/media/oss-project-sequence.svg)
+## Specifications
 
-To deliver on the Notary v2 goals of cross registry movement of artifacts with their signatures, changes to several projects are anticipated, including [OCI distribution-spec][oci-distribution-spec], [CNCF Distribution][cncf-distribution], [OCI Artifacts][oci-artifacts], [ORAS][oras] with further consumption from projects including [containerd][containerd], [OPA][opa], [Gatekeeper][gatekeeper] and the [docker client][docker-client].
+| File Name     | Description |
+| -------- | ----------- |
+| [plugin-extensibility.md](./specs/plugin-extensibility.md) | Notation Plugin specification |
+| [signature-envelope-cose.md](./specs/signature-envelope-cose.md) | Notary Project OCI COSE signature envelope |
+| [signature-envelope-jws.md](./specs/signature-envelope-jws.md) | Notary Project OCI JWS signature envelope |
+| [signature-specification.md](./specs/signature-specification.md) | Notary Project OCI signature specification |
+| [signing-and-verification-workflow.md](./specs/signing-and-verification-workflow.md) | Notary Project OCI signing and verification workflow |
+| [signing-scheme.md](./specs/signing-scheme.md) | Notary Project signing scheme|
+| [trust-store-trust-policy.md](./specs/trust-store-trust-policy.md) | Notation Trust Store and Trust Policy  |
 
-Notary v2 aims to solve the intra and cross registry signing & validating scenarios through the following prototypical experiences:
 
-**Docker build, sign, push, pull, verify**:
+## Threat Models
 
-```bash
-docker build -t registry.wabbit-networks.io/net-monitor:v1 .
+| File Name     | Description |
+| -------- | ----------- |
+| [notation-threatmodel.md](./threatmodels/notation-threatmodel.md) | Threat models for [Notation CLI](https://github.com/notaryproject/notation) |
```

Comment:
Should those detailed stuffs go to the `README.md` files in their corresponding folders?

For instance, the descriptions of each file under the `Requirements` section should be written in the `./requirements/README.md` file.

Otherwise, this `README.md` file will not be scalable and too long to read.

Reply:
Good suggestions. I think this work can be combined to issue https://github.com/notaryproject/.github/issues/48 once the new folder structure is decided.

Reply:
@yizha1 Could you create an issue for this?

Reply:
@shizhMSFT Done. Here it is https://github.com/notaryproject/notaryproject/issues/271